windows_containers/enabling-windows-container-workloads.adoc

```diff
@@ -20,18 +20,22 @@ Before adding Windows workloads to your cluster, you must install the Windows Ma
 
 * You are running an {product-title} cluster version 4.6.8 or later.
 
+.Additional resources
+* For the comprehensive prerequisites for the Windows Machine Config Operator, see xref:understanding-windows-container-workloads.adoc#wmco-prerequisites_understanding-windows-container-workloads[Understanding Windows container workloads].
```

Comment:
@EricPonvelle xref links MUST be the full path. This should be: 

xref:../understanding-windows-container-workloads.adoc#wmco-prerequisites_understanding-windows-container-workloads

Can you make the fix and CP to the right branches?

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

Reply:
My mistake! I figured since it was linking to a neighboring file it didn't need the full structure but noted!

Both fixes are merged in 

https://github.com/openshift/openshift-docs/pull/34153
https://github.com/openshift/openshift-docs/pull/34151